runtime/src/lib.rs

```diff
@@ -1965,6 +1972,7 @@ impl order_book::Config for Runtime {
     type EnsureTradingPairExists = TradingPair;
     type AssetInfoProvider = Assets;
     type DexInfoProvider = DEXManager;
+    type LockTechAccountId = GetOrderBookLockTechAccountId;
```

Comment:
@vovac12 What do you think, is it ok to have 1 tech account for all order books.
Mathematically it's ok, but we have the mixed liquidity of all order books in 1 account.

Reply:
It seems that having one for each trading pair or something similar is not possible, because of the complicated process of registering the accounts (and having to do runtime migrations for it). However yes, let's wait for Vladimirs answer, maybe I'm wrong

Reply:
You're right, it's better to have tech account for each trading pair. If we will register tech accounts during order book registration we will not need migrations at all. I think we should add new variant to `TechPurpose` enum and create order book tech account with it

Reply:
I suppose they also should be deregistered on removal of trading pair, right?